internal/acceptance/metastore_test.go

```diff
@@ -0,0 +1,41 @@
+package acceptance
+
+import (
+	"os"
+	"testing"
+)
+
+func TestUcAccMetastore(t *testing.T) {
+	cloudEnv := os.Getenv("CLOUD_ENV")
+	switch cloudEnv {
+	case "ucacct":
```

Comment:
Are these the right names? I think this should be `aws-prod-ucacct` and gcp should be `gcp-acct-prod`

Reply:
I looked it up from our TF env https://github.com/databricks/eng-dev-ecosystem/blob/main/ops/environments/aws-prod/aws-prod-ucacct.tf#L5